cluster/manifests/wiz/sensor/dameonset.yaml

```diff
@@ -0,0 +1,223 @@
+{{ if eq .Cluster.ConfigItems.wiz_enable_runtime_monitoring_daemonset "true"}}
+---
+# Source: wiz-sensor/templates/daemonset.yaml
+apiVersion: apps/v1
+kind: DaemonSet
+metadata:
+  name: wiz-sensor
+  labels:
+    
+    helm.sh/chart: wiz-sensor-1.0.4760
+    image/tag: v1
+    app.kubernetes.io/name: wiz-sensor
+    app.kubernetes.io/instance: wiz-sensor
+    app.kubernetes.io/version: "1.0.4708"
+    app.kubernetes.io/managed-by: Helm
+    application: "wiz"
+    component: "sensor"
+  namespace: wiz
+
+spec:
+  selector:
+    matchLabels:
+      app.kubernetes.io/name: wiz-sensor
+      app.kubernetes.io/instance: wiz-sensor
+
+  updateStrategy:
+    type: RollingUpdate
+
+  template:
+    metadata:
+      labels:
+        helm.sh/chart: wiz-sensor-1.0.4760
+        image/tag: v1
+        app.kubernetes.io/name: wiz-sensor
+        app.kubernetes.io/instance: wiz-sensor
+        app.kubernetes.io/version: "1.0.4708"
+        app.kubernetes.io/managed-by: Helm
+      annotations:
+        container.apparmor.security.beta.kubernetes.io/wiz-sensor: unconfined
+        cluster-autoscaler.kubernetes.io/enable-ds-eviction: "true"
+        node-ready.cluster.zalando.org/exclude: "true"
+
+    spec:
+      serviceAccountName: wiz-sensor
+      affinity:
+        nodeAffinity:
+            requiredDuringSchedulingIgnoredDuringExecution:
+              nodeSelectorTerms:
+              - matchExpressions:
+                - key: master
+                  operator: DoesNotExist
+              - matchExpressions:
+                - key: dedicated
+                  operator: NotIn
+                  values:
+                  - skipper-ingress
```

Comment:
We also have `skipper` and `skipper-ingress-redis` labelled nodes that we will have to skip if we're skipping `skipper-ingress`.

We also have other "system" dedicated nodes like `zmon-worker-arm64`, `zmon-redis` and `zmon-appliance` labelled nodes for ZMON.

Reply:
I'm assuming we are scoping the Wiz system only to "application worker nodes", please correct me if this isn't necessarily the case.